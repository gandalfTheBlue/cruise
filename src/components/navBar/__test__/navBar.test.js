import React from 'react'
import renderer from 'react-test-renderer'

import NavBar from '../navBar'

it('renders correctly nav bar content', () => {
  const tree = renderer.create(<NavBar />).toJSON()
  expect(tree).toMatchSnapshot()
})
