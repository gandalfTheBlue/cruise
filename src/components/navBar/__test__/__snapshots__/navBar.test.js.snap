// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly nav bar content 1`] = `
<div
  className="navbar"
>
  <ul />
</div>
`;
